scripts/deploysnapshot.sh

```diff
@@ -21,6 +21,7 @@ SNAPSHOTDIR=/var/www/vhosts/mf-geoadmin3/private/snapshots/$1
 # Prepare snapshot
 cd $SNAPSHOTDIR/geoadmin/code/geoadmin
 if [ "$NEW_VERSION" = "true" ]; then
+  export KEEP_VERSION='false'
```

Comment:
We actually always want keep the version when deploying the first time so `KEEP_VERSION='true'`. Optionally, to avoid overriding what we have in production we want to generate a new version from a snapshot for instance.
So behaviour is correct. Now we could remove the NEW_VERSION parameter that is correct. And simply do
`make s3deployint KEEP_VERSION='false'.